.emacs.d/lisp/user/machine-local.el

```diff
@@ -0,0 +1,5 @@
+(let ((dot-config (concat (file-name-directory (or (getenv "XDG_CONFIG_HOME") "~/.config/")) "dot/"))
```

Comment:
Just so that we are able to perhaps extend this functionality in the future, that is.